schedule/yast/btrfs/btrfs_opensuse.yaml

```diff
@@ -2,7 +2,7 @@ name:           btrfs
 description:    >
   Validate default installation with btrfs.
 vars:
-  FILESYSTEM: btrfs
+  FILESYSTEM: btrfs  # Used for log collection in lib/y2_base
```

Comment:
I'm not sure about the parser with comments on the rights, I guess it is fine but might give troubles sometimes, it is better on top of the line. Better way would be to create a ticket to remove this dependency  and being able to check if the filesystem is btrfs and avoid the comment. We could refine that of course.